src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -118,33 +121,31 @@
   grid-template-areas:
     "brand close"
     "header header"
-    "content content"
-    "logo logo";
+    "content content";
 
   // Modal is full screen until larger breakpoint
   width: 100vw;
   height: 100vh;
   overflow-x: hidden;
   overflow-y: auto;
+  /* stylelint-disable */
+  scrollbar-color: #c10;
   background-color: var(--pf-c-about-modal-box--BackgroundColor); // Because this component is always dark, set the background color
   box-shadow: var(--pf-c-about-modal-box--BoxShadow);
+  grid-template-columns: var(--pf-c-about-modal-box--grid-template-columns);
```

Comment:
this var isn't defined. should we define a default or just remove it? if we don't need it, I would probably just define `grid-template-columns` in the breakpoints